packages/driver/src/cypress/error_messages.ts

```diff
@@ -1710,6 +1710,32 @@ export default {
 
       This is likely because the data argument specified is not serializable. Note that functions and DOM objects cannot be serialized.`,
     },
+    callback_mixes_sync_and_async: {
```

Comment:
any suggestions on wording is very welcome 😅

Reply:
I wonder if something like `callback_mixes_sync_and_commands` is more appropriate? Not entirely sure 🤔